api/cpp/include/slint.h

```diff
@@ -129,6 +129,13 @@ inline void debug(const SharedString &str)
     cbindgen_private::slint_debug(&str);
 }
 
+constexpr inline int cast_float_to_int(float f)
+{
+    // Casting > int_max, etc. is UB, so clamp.
```

Comment:
Why not use std::clamp? Because of NaN?

Reply:
Just me being forgetful - thanks :). Maybe that also helps with the CI.